js-rattler/package.json

```diff
@@ -38,5 +38,8 @@
         "webpack": "^5.96.1",
         "webpack-cli": "^5.1.4"
     },
-    "type": "module"
+    "type": "module",
```

Comment:
this is wrong, we build umd output so this option should not be set.

Reply:
👍